models/repo.go

```diff
@@ -1992,6 +1992,17 @@ func DeleteRepository(doer *User, uid, repoID int64) error {
 		}
 	}
 
+	releaseAttachments := make([]string, 0, 20)
+	attachments := make([]*Attachment, 0, len(releaseAttachments))
```

Comment:
```suggestion
	attachments := make([]*Attachment, 0, cap(releaseAttachments))
```
Or better, move this to after `releaseAttachments ` is populated.

Reply:
@guillep2k attachments is populated by xorm before releaseAttachments. 
Maybe you mean :
```
attachments := make([]*Attachment, 0, 20)
if err = sess.Join("INNER", "release", "release.id = attachment.release_id") ... {
...
}
releaseAttachments := make([]string, len(attachments))
for i := 0; i < len(attachments); i++ {
	releaseAttachments = append(releaseAttachments, attachments[i].LocalPath())
}
```